core/backend/ios/core-test-runner/core-test-runner.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "itwin-mobile-native-ios",
+        "repositoryURL": "https://github.com/iTwin/mobile-native-ios",
+        "state": {
+          "branch": null,
+          "revision": "d1f5b600437a990f408980ebab36ffb049aac85f",
+          "version": "3.1.6"
```

Comment:
does this file need to be committed? Wondering what the workflow will be for updating this.

Reply:
I removed this file, we might want to add Package.resolved files to the .gitignore.